tests/_files/source_with_multiline_constant_return.php

```diff
@@ -396,7 +396,7 @@ public function unaryLogicalNotWithNotConstInTheMiddle(): string
     public function unaryMinusWithNotConstInTheMiddle(): string
     {
         return -
-        (
+        (int) (
```

Comment:
cast is tested in a following method, this method should test "unary minus with not const in the middle"

Reply:
We need valid source code for proper debugging, and this doesn't alter the validity of the test

Reply:
it alters - unary minus of concat (const expr), not unary minus of cast expr

Reply:
It makes no sense to test something that doesn't even compile.

Reply:
it compiles fine: https://3v4l.org/aaq3l

Reply:
Indeed this work, but the other doesn't